Frequency/Board/Buildings/Building2.py

```diff
@@ -0,0 +1,17 @@
+class Building:
+
+    def __init__(self, player, tile, texture):
+        self.Player = player
+        self.Tile = tile
+        self.Texture
+
+
+    def Update(self, game):
+        return self
+
+
+    def Draw(self, game):
+        tileSize = min(game.Settings.Resolution.X // game.Settings.GetMaxTiles().X,
```

Comment:
Je hebt ook toegang tot de tile. Kan je daar niet gewoon de maat van nemen in plaats van hier opnieuw bereken